MatrixSDK/Space/MXSpaceChildInfo.swift

```diff
@@ -52,14 +55,28 @@ public class MXSpaceChildInfo: NSObject {
     public let activeMemberCount: Int
     
     /// Allows a space admin to list the sub-spaces and rooms in that space which should be automatically joined by members of that space.
-    public let autoJoin: Bool
+    public var autoJoin: Bool
+    
+    /// `true` if the room is suggested. `false` otherwise.
+    public var suggested: Bool = false
+    
+    /// List of space parent IDs
+    public var parentIds: Set<String> = Set()
+    
+    /// List of children IDs
+    public var childrenIds: [String] = []
```

Comment:
It can be safer if we can avoid these immutable variables and put default values in the initializer. In order to have read only objects and prevent modification from the outside of the SDK.